app/marketing/mails.py

```diff
@@ -122,6 +124,57 @@ def new_grant(grant, profile):
         translation.activate(cur_language)
 
 
+def change_grant_owner_request(grant, profile):
+    from_email = settings.CONTACT_EMAIL
+    to_email = profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_change_grant_owner_request(grant)
+
+        if not should_suppress_notification_email(to_email, 'change_owner'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
```

Comment:
Why not `except` ? You should catch the error and log in the error logger. So we can track what is wrong if need be.